react-app/src/utils/leaguehelpers.js

```diff
@@ -74,10 +81,48 @@ export default {
 
 	convertMatchTime: (time) => {
 		//This is just for the schedules bc theyre always at night
+		//we can change it later..
```

Comment:
Yeah I have to rework the dates so that you can use js built in Date objects.. not something that will get fixed this summer